yaml-tests/src/test/java/MultiServerJDBCYamlTests.java

```diff
@@ -30,8 +31,11 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
+import static com.apple.foundationdb.relational.yamltests.YamlExecutionContext.OPTION_FORCE_CONTINUATIONS;
```

Comment:
Given this is only used once, I _feel_ like it would be better to not have a static import for it.
But I also suggested removing this in favor of an object you can call methods on.

Reply:
Done.